cmake/Modules/FindNCCL.cmake

```diff
@@ -33,6 +33,26 @@
 
 set(NCCL_ROOT_DIR "" CACHE PATH "Folder contains NVIDIA NCCL")
 
+# if CUDAToolkit_FOUND is not found, try default location
+#if (NOT CUDAToolkit_FOUND)
+  if (UNIX)
+    set (search_paths "/usr/local/cuda")
+
+    find_path(NCCL_INCLUDE_DIRS
+    NAMES nccl.h
+    PATHS ${search_paths}
```

Comment:
Should this be HINTS?

Reply:
https://cmake.org/cmake/help/v3.4/command/find_path.html
Looks like either Hints or paths is permitted. Any specific reason to use hints?

Reply:
When writing the comment I thought PATHS may skip default locations whereas HINTS would be defaults + specified locations. Thanks for pointing out they are equivalent.